create-projects/create-cd-jenkins.sh

```diff
@@ -1,83 +1,110 @@
 #!/usr/bin/env bash
 set -eu
 
-# This script sets up a ${PROJECT_ID}-cd/jenkins-master and the associated webhook proxy for the project you are creating.
+# This script sets up a ${PROJECT_ID}-cd/jenkins-master and the associated
+# webhook proxy for the project you are creating.
 
 TAILOR="tailor"
-VERBOSE=false
-STATUS=false
+ODS_NAMESPACE="ods"
+ODS_IMAGE_TAG="lastest"
+PROJECT_ID=""
```

Comment:
I would not mind having the ability to pass parameters via environment variables so one would not have to explicitly pass them into tailor. 
On the other hand this makes it less explicit on where parameters are used.

Reply:
Hmm ... I decided to reset everything as those scripts are executed within the context of Jenkins, which has some env vars exposed (via the `DeploymentConfig`, but also from inside the image). It might be surprising to have them alter what the script does without seeing them passed/set in the `Jenkinsfile`. That's why I opted to have all params explicitly, and prevent any effect of outside env vars.

Reply:
I can live with that as well. Perhaps we could add a comment in the code so that we don't forget about this later. Or perhaps better add a test that would fail if someone makes changes like that.

Reply:
Cool. Added comments in 1ff8267.